code/x86-intel/hello-world/hello-world-macos.asm

```diff
@@ -1,6 +1,6 @@
 ;; Hello World in x86 Intel syntax
 
-;; Intel mac assembly instructions
+;; Intel MacOS assembly instructions
```

Comment:
Mind updating the name to `hello-world-mac` and the commented running instructions in here to be `hello-world-mac.asm`?